egs/wsj/s5/steps/libs/nnet3/train/frame_level_objf/common.py

```diff
@@ -30,7 +29,7 @@ def train_new_models(dir, iter, srand, num_jobs,
                      shuffle_buffer_size, minibatch_size,
                      cache_read_opt, run_opts,
                      frames_per_eg=-1,
-                     min_deriv_time=None, max_deriv_time=None):
```

Comment:
I had to reload the page, github was displaying it wrong.  But there is another instance in this script where the same issue exists.  I don't want to have to check it myself for errors like this.  Can you please go through every line of your changes to common scripts?  Bugs in these will be particularly inconvenient to deal with.